Modules/_multiprocessing/multiprocessing.h

```diff
@@ -32,8 +32,12 @@
 #  define BOOL int
 #  define UINT32 uint32_t
 #  define INT32 int32_t
-#  define TRUE 1
-#  define FALSE 0
+#  ifndef TRUE
+#    define TRUE 1
+#  endif
+#  ifndef FALSE
+#    define FALSE 0
+#  endif
```

Comment:
Doesn't look correct to me.  multiprocessing requires TRUE = 1.  I think you should 'undef' the existing TRUE/FALSE and define what multiprocessing expects.